app/editor/src/features/admin/reports/constants/defaultReportSection.ts

```diff
@@ -4,7 +4,7 @@ export const defaultReportSection = (reportId: number): IReportSectionModel => {
   return {
     id: 0,
     reportId: reportId,
-    name: '',
+    name: crypto.randomUUID(),
```

Comment:
Need a static naming convention for sections as the name is used to link the content to a section within the template.